### DISCUSSION TITLE
How do I move to a new system

### DISCUSSION THREAD

**Discussion by widert** [zwave-js/node-red-contrib-zwave-js#293] [Q&A]
I'm moving my complete node-red installation (including my z-wave dongle) to a new system, what is the easiest way to do this?

**Comment by marcus-j-davies** [comment]
Hi @widert 

To Move Node RED over:
 - Copy everything in `~/.node-red` **APART FROM** `~/.node-red/node_modules` <- this folder is machine/architecture  specific
 - Paste its contents in the new `~/.node-red` (on the new machine)
 - CD into  `~/.node-red` (on the new machine)
 - Issue `npm install` (recreates `~/.node-red/node_modules` for the machine that will run your Node RED flows)
 
 This is the normal move routines (and one often suggested on the Node RED forums)
 As you're  copying `./.node-red` - It will also copy your ZWave Network data files
   - For the record, this is  `./.node-red/zwave-js-cache`
